table_connector.py

```diff
@@ -83,9 +83,12 @@ def __get_address_joined_with_message_id(address_book):
     )
     address_joined_with_message_id = address_joined_with_message_id.drop(['ROWID'], axis=1)
 
-    # Confirm that message_id, chat_id and phone_or_email form a composite key
-    assert sum(address_joined_with_message_id.duplicated(subset=['message_id', 'chat_id', 'phone_or_email'],
-                                                         keep=False)) == 0
+    key_fields = ['message_id', 'chat_id', 'phone_or_email']
+    duplicates = address_joined_with_message_id.duplicated(subset=key_fields,
+                                                           keep=False)
+    error_message = ("message_id, chat_id and phone_or_email "
+                     "do not form a composite key")
+    assert sum(duplicates) == 0, error_message
```

Comment:
At this point you make me think we should just print an error and continue...Can you switch to doing that?  Technically nothing will break, just some counts later on will be off